src/filters/read.rs

```diff
@@ -33,6 +33,7 @@ pub struct ReadContext {
     pub endpoints: Arc<ClusterMap>,
     /// The upstream endpoints that the packet will be forwarded to.
     pub destinations: Vec<Endpoint>,
+    pub faster_destinations: Vec<EndpointAddress>,
```

Comment:
Can we give this a more descriptive name and a documentation comment of how it gets populated and why it is used?

Reply:
Oh yah, this was just temporary naming, I'll fix.